src/Committee/CommitteeManager.php

```diff
@@ -479,4 +479,34 @@ public function getCommitteeSupervisor(Committee $committee): ?Adherent
     {
         return $this->getMembershipRepository()->findSupervisor($committee->getUuid()->toString());
     }
+
+    public function getCommitteesAndMembersByCoordinates(
+        Coordinates $coordinates,
+        int $count = self::COMMITTEE_PROPOSALS_COUNT
+    ): array {
+        $committees = $this->getCommitteeRepository()->findNearbyCommittees($coordinates, $count);
+
+        return  $this->getFormattedCommitteesAndMembers($committees);
+    }
+
+    public function getLastApprovedCommitteesAndMembers(int $count = self::COMMITTEE_PROPOSALS_COUNT): array
+    {
+        $committees = $this->getCommitteeRepository()->findLastApprovedCommittees($count);
+
+        return $this->getFormattedCommitteesAndMembers($committees);
+    }
+
+    private function getFormattedCommitteesAndMembers(array $committees): array
```

Comment:
I think you can remove this method, and call the `getMembersCount` on each committee in `CommitteeAroundAdherentType::finishView`